apps/openmw/mwworld/cellref.hpp

```diff
@@ -34,6 +34,9 @@ namespace MWWorld
         // Id of object being referenced
         std::string getRefId() const;
 
+        // Pointer to ID of the object being referenced
+        const std::string* getRefIdPtr() const;
```

Comment:
Any reason why it's a pointer instead of a reference? This function can't return `nullptr`.

Reply:
As stated in the description that was the suggestion. It hasn't been given much more of a thought.

Reply:
Then it should be a reference at least. Still, pretty hacky. As a workaorund for now maybe okay. As a long term solution not so much. Scrawl and I once discussed a plan to implement something like unreal engines FName (https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/UnrealArchitecture/StringHandling/FName/) for IDs. I guess that never materialised?

Reply:
@zinnschlag There is a MR to optimise this: https://gitlab.com/OpenMW/openmw/-/merge_requests/1160

Moving away from this however is not in the issue tracker is it?

Reply:
> @zinnschlag There is a MR to optimise this: https://gitlab.com/OpenMW/openmw/-/merge_requests/1160
> 
> Moving away from this however is not in the issue tracker is it?

Not that I know of. This would require a fair bit of design work. Getting something like FName right isn't a simple task.